src/cljs/main/broadfcui/utils.cljs

```diff
@@ -103,22 +103,12 @@
     "local.broadinstitute.org"
     (string/join "." (rest (string/split js/window.location.hostname ".")))))
 
-(defn get-leo-cookie-domain []
```

Comment:
`get-leo-cookie-domain` and `set-notebooks-access-token-cookie` are no longer used